dev/puppet-server.conf.sample

```diff
@@ -115,3 +115,8 @@ authorization: {
         }
     ]
 }
+
+# general puppetserver settings
+puppetserver: {
+    environment-class-cache-enabled: true
+}
```

Comment:
Huh, okay. I guess I've just sort of always thought that the name of that section is weird and renamed it in my head to "puppetserver", and so I find having a separate section more confusing than having this setting in a JRuby section. I don't think at all that I would think JRuby had nothing to do with caching of environments.

Have you asked anyone in CS about this? If you have then I'll shut up. :)